models/classes/search/SearchProxy.php

```diff
@@ -165,6 +168,25 @@ public function removeFromGenerisSearchWhiteList(array $whiteList): void
         );
     }
 
+    /**
+     * Gets the whitelist of RDF URLs that can be used with Advanced Search.
+     *
+     * @return string[]
+     */
+    public function getWhitelist(): array
```

Comment:
- [ ] Should not we use this method on `isForcingDefaultSearch` now? https://github.com/oat-sa/tao-core/pull/3357/files#diff-11e00d06172cc2f0ee040827cf177ec12219e921c93b53eddf42173fac863cb9L219

- [ ] Maybe we could provide a more self explainable name? I saw you wrote a comment in the method, probably because the name is not clear.

```suggestion
    public function getGenerisSearchUriWhitelist(): array
```

Reply:
Renamed in f5357c03731dbeffc7fa943c97ba1f2f21863a99

Reply:
First point already done in a497f64bf09233b615b31464a18a6a2bd096045f